### PR TITLE
chore(localizations): Update `nl-NL`

### PR DESCRIPTION
**Pull request by guustgoossens**
## Description
Completed nl-NL localization

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: .changeset/rude-beds-rest.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/localizations': minor
```
Comment (alexcarpenter) [inline .changeset/rude-beds-rest.md:2]:
```suggestion
'@clerk/localizations': patch
```

File: .changeset/rude-beds-rest.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Improved nl-NL localization
```
Comment (alexcarpenter) [inline .changeset/rude-beds-rest.md:5]:
```suggestion
Improve `nl-NL` localization values.
```

File: package.json [hunk 3]
```diff
@@ -55,7 +55,7 @@
   "devDependencies": {
     "@actions/core": "^1.10.1",
     "@arethetypeswrong/cli": "^0.15.3",
-    "@changesets/cli": "^2.26.2",
+    "@changesets/cli": "^2.27.8",
```
Comment (alexcarpenter) [inline package.json:58]: @guustgoossens can you revert this change please?

**Review by alexcarpenter — COMMENTED** [review]
Looks good, lets just revert the change in package.json.

Will close https://github.com/clerk/javascript/pull/4116 in favor of this one.

File: package.json (hunk 3 above)
Comment (guustgoossens) [reply package.json]: @alexcarpenter done!

Comment (alexcarpenter) [reply package.json]: You'll need to run an `npm i` to remove the changes in the package-lock file as well.

Comment (guustgoossens) [reply package.json]: @alexcarpenter just did that, thanks for the patience.

Comment (alexcarpenter) [reply package.json]: @guustgoossens no problem at all! Still seeing changes to the package-lock.json file. Can you revert those changes please?

**Review by tmilewski — CHANGES_REQUESTED** [review]
@guustgoossens Mind resolving the PR conflicts? I tried doing it myself but, despite allowing "maintainer edits," the permissions still seem off.

It might make sense to completely remove the `package-lock.json` updates, just for the sake of this PR.

Thanks a lot!
